package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "angular-bootstrap",
+    "version": "0.12.0",
+    "description": "Native AngularJS (Angular) directives for Bootstrap.",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/angular-ui/bootstrap-bower"
+    },
+    "author": {
+        "name": "https://github.com/angular-ui/bootstrap/graphs/contributors"
+    },
+    "keywords": [
+        "angular",
+        "angular-ui",
+        "bootstrap"
+    ],
+    "license": "MIT",
+    "ignore": [],
+    "main": "./ui-bootstrap-tpls.js",
+    "dependencies": {
+        "angular": ">=1 <1.4.0",
+        "bootstrap": ">=3 <4"
```

Comment:
It's functionally equivalent, but your use case is exactly why the shorthands exist.  Also, it's the format that is written when `npm install --save` is run.  

That format works for the Angular dependency as the other way to write it would be `~1.0 | ~1.1 | ~1.2 | ~1.3`, which is more verbose.